Example/Source/Examples/HorizontalScroll/HorizontalCollectionViewController.swift

```diff
@@ -72,9 +72,9 @@ extension HorizontalCollectionViewController: BrickRepeatCountDataSource {
 
 extension HorizontalCollectionViewController: CollectionBrickCellDataSource {
 
-    func configureCollectionBrickViewForBrickCollectionCell(brickCollectionCell: CollectionBrickCell) {
-        brickCollectionCell.brickCollectionView.registerBrickClass(LabelBrick.self)
-        brickCollectionCell.brickCollectionView.registerBrickClass(ImageBrick.self)
+    func registerBricks(for cell: CollectionBrickCell) {
+        cell.brickCollectionView.registerBrickClass(LabelBrick.self)
```

Comment:
As you are registering just the class, shouldn't you add these brickTypes in the initializer of the CollectionBrick?